### ISSUE TITLE
Players looking backwards or different direction

### ISSUE THREAD

**Issue by userunknown198923** [nullworks/cathook#960]
Most painfull when u are playing sniper and missing headshot

Players are just looking backwards or random location i will try send picture of it.
(Im using ubuntu)

**Comment by LightPower392** [comment]
Is this referring to anti-aim?
Are you asking how to hvh?

**Comment by BenCat07** [comment]
Some people seem to have a bug where some players look backwards insteas of forward. I cannot identify what causes this bug, and it only seems to happen to a select few people

**Comment by TDiffff** [comment]
That also happened to me once or twice. It's very rare for my case.

**Comment by Short-TermWreck91** [comment]
yeah same here, sometimes its just looking for random direction, its glitch i think

**Comment by TotallyNotElite** [comment]
How to reproduce?

**Comment by BenCat07** [comment]
you don't, that's the problem

**Comment by Short-TermWreck91** [comment]
its maybe anti-cheat when it detects aimbot or antiaim idk

**Comment by BenCat07** [comment]
anticheat does nothing with angles, antiaim only affects local player

**Comment by anonymousguy42452** [comment]
it can be maybe graphics problem or yaw resolver its really anoying
